src/pages/Menu/Menu.tsx

```diff
@@ -31,16 +30,14 @@ export const MenuPage = (): ReactElement => {
       <Helmet>
         <title>Menu - Answer King</title>
       </Helmet>
-      <div className="menu">
+      <div className="">
         <MenuCategories categories={categories.data} />
 
-        <div className="menu__items">
+        <div className="">
```

Comment:
are these `classNames` supposed to be empty? can remove them if so

Reply:
Ah yeah I started styling this just as we decided to split it up into other tickets. I'll remove them just for the sake of keeping the branch clean but I guess ultimately it doesn't matter too much as we have a ticket in play which will overwrite this anyway 👍